sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientTest.java

```diff
@@ -46,20 +53,27 @@ static void afterAll() {
         StepVerifier.resetDefaultTimeout();
     }
 
-    @Override
-    protected void beforeTest() {
-        client = clientSetup(httpPipeline -> new FormRecognizerClientBuilder()
```

Comment:
Should we be good to delete `clientSetup` from FomRecognizerClientTestBase now .